src/dns_decode.erl

```diff
@@ -624,6 +624,15 @@ decode_rrdata(
         iterations = Iterations,
         salt = Salt
     };
+decode_rrdata(
+    _MsgBin, _Class, ?DNS_TYPE_TLSA, <<Usage:8, Selector:8, MatchingType:8, CAData/binary>>
+) ->
+    #dns_rrdata_tlsa{
+        usage = Usage,
+        selector = Selector,
+        matching_type = MatchingType,
+        certificate_association_data = CAData
```

Comment:
While the RFC section refers to the "Certificate Association Data", I think we can probably get away with a simpler "certificate". Both are used interchangibly, and in most implementation I've seen it's called certificate.

(similarly, certificate usage is called "usage" as you've done here).